.travis.yml

```diff
@@ -5,8 +5,11 @@ sudo: required
 services:
   - docker
 
+# install latest rebar3 because the one travis provides doesn't support cover_export_name option
 before_install:
+  - git clone https://github.com/erlang/rebar3.git; cd rebar3; ./bootstrap; sudo mv rebar3 /usr/local/bin/; cd ..
```

Comment:
Would be nice to checkout certain commit instead of master

Reply:
this is only used in travis. no big deal if it breaks.